packages/next/src/client/components/react-dev-overlay/internal/container/RuntimeError/index.tsx

```diff
@@ -196,7 +198,7 @@ export const styles = css`
   [data-nextjs-collapsed-call-stack-details] summary {
     display: flex;
     align-items: center;
-    margin: var(--size-gap-double) 0;
+    margin-bottom: var(--size-gap);
```

Comment:
Fixed some inconsistent spacing, here is the before/after. (it makes the stack shorter as well):
![image](https://github.com/vercel/next.js/assets/18369201/fb680b88-aa22-4be0-9774-25dd15021e40) ![image](https://github.com/vercel/next.js/assets/18369201/241e7975-7b89-42d6-bab3-b1e3a35e10c7)